packages/tooltips/src/elements/Tooltip.tsx

```diff
@@ -40,43 +43,54 @@ const TooltipWrapper = styled.div`
   z-index: ${props => props.zIndex};
 `;
 
-const SIZE = {
-  SMALL: 'small',
-  MEDIUM: 'medium',
-  LARGE: 'large',
-  EXTRA_LARGE: 'extra-large'
-};
-
-const TYPE = {
-  LIGHT: 'light',
-  DARK: 'dark'
-};
+type TOOLTIP_TYPE = 'light' | 'dark';
 
-/**
- * Due to Popper.JS needing a reference to a component we provide a simple wrapper
- * to ensure the correct reference is provided.
- */
-const TriggerWrapper = styled.div`
-  display: inline-block;
-`;
-
-const Tooltip = ({
-  appendToBody,
-  arrow,
-  children,
-  delayMilliseconds,
-  eventsEnabled,
+export interface ITooltipProps
+  extends Partial<ThemeProps<DefaultTheme>>,
+    React.HTMLAttributes<HTMLDivElement> {
+  /** Appends the tooltip to the body element */
+  appendToNode?: Element;
+  hasArrow?: boolean;
+  /** Milliseconds of delay before open/close of tooltip is initiated */
+  delayMilliseconds?: number;
+  /** Whether Popper.js should update based on DOM resize events */
+  eventsEnabled?: boolean;
+  id?: string;
+  content: React.ReactNode;
+  /**
+   * These placements differ from the default naming of Popper.JS placements to help
+   * assist with RTL layouts.
+   **/
+  placement?: GARDEN_PLACEMENT;
+  /** Passes options to [Popper.JS Instance](https://github.com/FezVrasta/popper.js/blob/master/docs/_includes/popper-documentation.md#new-popperreference-popper-options) */
+  popperModifiers?: any;
```

Comment:
Do you think the [`Modifiers` interface](https://github.com/FezVrasta/popper.js/blob/master/packages/popper/index.d.ts#L56) from PopperJS could be used here as the type?

Reply:
I originally saw this wasn't exported from the `react-popper` package so I didn't use it. We should be ok since PopperJS is a `dependency`. Added.